src/nnet3/nnet-training.cc

```diff
@@ -260,7 +258,7 @@ void ObjectiveFunctionInfo::UpdateStats(
     BaseFloat this_minibatch_tot_aux_objf) {
   int32 phase = minibatch_counter / minibatches_per_phase;
   if (phase != current_phase) {
-    KALDI_ASSERT(phase == current_phase + 1); // or doesn't really make sense.
+    //KALDI_ASSERT(phase == current_phase + 1); // or doesn't really make sense.
```

Comment:
have to comment it out, since otherwise the assert will fail because of the adversarial training with probability